### ISSUE TITLE
Add L1-regularized SVM and c5.0 or fast frugal trees

### ISSUE THREAD

**Issue by BTopcuoglu** [SchlossLab/ML_pipeline_microbiome#94]
- Create new models to be used
- Test with real data

**Comment by JMAStough** [comment]
I'd like to help with this if it's cool with yall

**Comment by kelly-sovacool** [comment]
> I'd like to help with this if it's cool with yall

@JMAStough Awesome! Sorry we didn't notice your comment until just now. I believe these would be custom models to implement, see caret docs here: https://topepo.github.io/caret/using-your-own-model-in-train.html.
Maybe @BTopcuoglu knows more about the details.

Let us know if you have any questions about the newly refactored code base.

**Comment by kelly-sovacool** [comment]
Note: @zenalapp is working on L1 logit for #154

**Comment by kelly-sovacool** [comment]
L1 logistic regression is now supported since we switched to `glmnet`. We should probably document which custom hyperparameters the user needs to provide to pick L1 vs L2.

**Comment by zenalapp** [comment]
Yes I think that should be in the hyperparameter tuning vignette.
